test/e2e/framework/create.go

```diff
@@ -300,6 +302,20 @@ func (f *Framework) PatchNamespace(item *string) {
 	}
 }
 
+// PatchContainerImages replaces the specified Container Registry with a custom
+// one provided via the KUBE_TEST_REPO_LIST env variable
+func (f *Framework) PatchContainerImages(containers []v1.Container) error {
```

Comment:
@oomichi: I think it makes sense to export it. The load/patch/create cycle intentionally exposes each step via an individual call because then patching can be done differently if desired, and then having the individual building blocks (like patching the namespace, patching container images) for such a patch method available makes sense.

Reply:
@pohly Thanks for your explanation, that makes sense :)

Reply:
Move this under https://github.com/kubernetes/kubernetes/tree/master/test/e2e/framework/pod and then lgtm.

Reply:
@timothysc I moved it under https://github.com/kubernetes/kubernetes/tree/master/test/e2e/framework/pod if you want to take a look again.